scripts/experiment_manager.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+#pylint: disable=unused-import, unused-variable, unused-argument
+# remove these disables once implemented
+'''
+This utility creates a command line interface for managing an experiment.
+'''
+
+import argparse
+import inspect
+import os
+import sys
+
+from uwtools.drivers import experiment
+from uwtools.utils import cli_helpers
+
+
+def parse_args(argv):# pragma: no cover
+    #remove pragma when completed
+
+    '''
+    Function maintains the arguments accepted by this script. Please see
+    Python's argparse documentation for more information about settings
+     of each argument.
+    '''
+    parser = argparse.ArgumentParser(
+       description='Set config with user-defined settings.'
+    )
+    parser.add_argument(
+        '--forecast_app',
+        help='If provided, will define the app to be run.\
+            Currently accepts SRW, MRW, or HAFS',
+        choices=["SRW", "MRW", "HAFS"],
+    )
+    parser.add_argument(
+        '-v', '--verbose',
+        action='store_true',
+        help='If provided, print all logging messages.',
+        )
+    parser.add_argument(
+        '-q', '--quiet',
+        action='store_true',
+        help='If provided, print no logging messages',
+        )
+    parser.add_argument(
+        '-l', '--log_file',
+        help='Optional path to a specified log file',
+        default=os.path.join(os.path.dirname(__file__), "forecast.log")
+        )
+    return parser.parse_args(argv)
+
+
+def manager(argv): # pragma: no cover
+    '''
+    Defines the user interface for the experiment manager. Parses arguments
+    provided by the user and passes to the facade to be run.'''
+    user_args = parse_args(argv)
+
+    # Set up logging
+    name = f"{inspect.stack()[0][3]}"
+    log = cli_helpers.setup_logging(user_args, log_name=name)
+
+    experiment_class = getattr(experiment, f"{user_args.forecast_app}Experiment")
+    managed_experiment = experiment_class(user_args.config_file, log_name=name)
```

Comment:
The signature of e.g. `SWRExperiment`'s constructor is `__init__(self, argv)`. Is that one of the classes that we'd be instantiating here? If so, `argv` doesn't seem like the right variable name on the constructor side, as these are already-parsed arguments (`argv` indicates raw, unparsed args); and it doesn't accept a second argument. The linter can't help us here because of the dynamic class lookup we're doing; however, when we add `mypy` type annotations (I'm working on a demo now), we could have something like in this `testing.py`:
```
from typing import Type

class Base:
    def __init__(self, n: int) -> None:
        self.n = n

class Sub(Base):
    pass

the_class: Type[Base] = globals()["Sub"]  # similar to getattr()
s = the_class(88, "foo")
```
and
```
% mypy testing.py 
testing.py:13: error: Too many arguments for "Base"  [call-arg]
Found 1 error in 1 file (checked 1 source file)
```
`pylint` and `mypy` both have to do static analysis, and `pylint` has no way of knowing what class we're going to end up instantiating. But via type annotations, `mypy` knows that we're going to have a subclass of `Base`, which has exactly one argument.

Reply:
Oh thats thoughtful - git really needs a 'lightbulb' react! I'm still learning so thats great, thanks

Reply:
Just to be clear, my only ask on this topic is that we either remove the `log_name=name` argument from the caller (here), or add a receiving argument to `Facade` and its subclasses. The rest is ideas for future work.

Reply:
Gotcha - just removed log_name=name for now to leave the rest for the future